system-addon/content-src/components/Search/_Search.scss

```diff
@@ -16,13 +15,15 @@
   width: 100%;
 
   input {
-    background: var(--newtab-textbox-background-color);
+    background: var(--newtab-textbox-background-color) var(--newtab-search-icon) $search-icon-padding center / $search-icon-size no-repeat;
```

Comment:
Oh oops! Almost forgot that the icon was centered before in the label, but now this is anchored to the left offset by $padding. For rtl, we'll want to offset from the right with the 2-value form: `background-position-x: right $search-icon-padding;`

<img width="251" alt="image" src="https://user-images.githubusercontent.com/438537/38982160-f9ddfcf6-4375-11e8-810e-c1ce756ec601.png">